languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -35382,7 +35382,21 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TOURS_CRITIQUES" name="Tours critiqués divers (barbarismes, impropriétés, solécismes, etc.)">
-    <rulegroup id="POUR_PAS_QUE" name="familier : pour (ne) pas que...">
+      <rule id="JE_M_EXCUSE " name="Je m’excuse" type="style" default="temp_off">
+          <pattern>
+              <token postag="SENT_START"/>
+              <marker>
+                  <token inflected="yes">Je</token>
+                  <token>m'</token>
+                  <token>excuse</token>
+              </marker>
+          </pattern>
```

Comment:
This pattern will also match "Je m'excuse toujours lorsque j'ai tort."

This could be fixed by adding "Je demande pardon", but the challenge is the Adverb which would then have to be placed between "demande" and "pardon". How about splitting the rule into two?